samples/web/send-from-aspnetcore-webapi/sample.md

```diff
@@ -2,88 +2,20 @@
 title: Using NServiceBus in an ASP.NET Core WebAPI Application
 summary: Illustrates how to send messages to a NServiceBus endpoint from a ASP.NET Core WebAPI application.
 component: Core
-reviewed: 2016-09-27
+reviewed: 2017-04-27
 related:
 - nservicebus/hosting
 ---
 
 
 This sample shows how to send messages to an NServiceBus endpoint from an ASP.NET Core WebAPI application. 
 
-WARNING: This sample runs on the full .NET framework 4.6.1 but utilizes both .NET Core and .NET Framework dependencies, i.e. ASP.NET Core and NServiceBus. So while .NET Core dependencies can be consumed by the .NET Framework runtime, the .NET Core runtime is not currently supported.
+WARNING: This sample runs on the full .NET Framework 4.6.1 but utilizes both .NET Standard and .NET Framework dependencies, i.e. ASP.NET Core and NServiceBus. So while .NET Standard dependencies can be consumed by the .NET Framework runtime, the .NET Core runtime is not currently supported.
```

Comment:
Fair point. I will add information about VS 2017 to pre-reqs and also to the info about the project template that was used.